.github/workflows/main.yml

```diff
@@ -18,9 +18,11 @@ jobs:
       matrix:
         node-version: [14.x]
     steps:
-    - uses: actions/checkout@v2
+      # https://github.com/actions/checkout/releases/tag/v4.0.0
+    - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
```

Comment:
Upgraded from `v2` to `v4` because this new version has already been tried and tested on `e2e.yml`

https://github.com/HathorNetwork/hathor-wallet/blob/503d91c701e1a98a358c7a11a426f16cd2a5d0b2/.github/workflows/e2e.yml#L21-L22